packages/vite-plugin-svelte-inspector/package.json

```diff
@@ -41,7 +41,7 @@
   },
   "peerDependencies": {
     "@sveltejs/vite-plugin-svelte": "^2.2.0",
-    "svelte": "^3.54.0",
+		"svelte": "^3.54.0 || ^4.0.0-next.0 || ^4.0.0",
```

Comment:
it looks like the rest of this file is using spaces, so this line is not aligned